_data-prepper/pipelines/expression-syntax.md

```diff
@@ -2,281 +2,181 @@
 layout: default
 title: Expression syntax
 parent: Pipelines
-nav_order: 12
+nav_order: 5
 ---
 
-# Expression syntax 
+# Expression syntax  
 
-The following sections provide information about expression syntax in Data Prepper.
+Expressions provide flexibility in data manipulation, filtering, and routing in Data Prepper. The following sections provide information about expression syntax in Data Prepper.
 
-## Supported operators
+## Key terms
 
-Operators are listed in order of precedence (top to bottom, left to right).
+The following key terms are used in the context of expressions:
 
-| Operator             | Description                                           | Associativity |
-|----------------------|-------------------------------------------------------|---------------|
-| `()`                 | Priority Expression                                   | left-to-right |
-| `not`<br> `+`<br>  `-`| Unary Logical NOT<br>Unary Positive<br>Unary negative | right-to-left |
-| `<`, `<=`, `>`, `>=` | Relational Operators                                  | left-to-right |
-| `==`, `!=`           | Equality Operators                                    | left-to-right |
-| `and`, `or`          | Conditional Expression                                | left-to-right |
-
-## Reserved for possible future functionality
-
-Reserved symbol set: `^`, `*`, `/`, `%`, `+`, `-`, `xor`, `=`, `+=`, `-=`, `*=`, `/=`, `%=`, `++`, `--`, `${<text>}`
-
-## Set initializer
+Term | Definition
+-----|-----------
+**Expression** | A generic component that contains a primary or an operator. Expressions can be nested withing other expressions. An expression's imminent children can contain 0–1 operators.
+**Expression string** | The highest priority in a Data Prepper expression and supports only one expression string that results in a return value. An expression string is not the same as an expression.
```

Comment:
A "primary" is defined below, as is an "operator." It would make sense to italicize those.

It may help to see how it is formatted in the original source. You'll see they italicize these terms and that makes it clearer.

https://github.com/opensearch-project/data-prepper/blob/main/docs/expression_syntax.md#definitions

Reply:
This is the priority of evaluating the expression tree when parsing.